scripts/run_benchmarks.sh

```diff
@@ -15,7 +15,7 @@ export xcm_generic_extrinsic="report_holding, buy_execution, query_response, tra
 
 cargo build --profile release \
     --features=runtime-benchmarks \
-    --manifest-path=node/Cargo.toml 
+    --manifest-path=node/Cargo.toml || exit -1
```

Comment:
Would you want to use `set -e` at the top instead?

Reply:
I usually don't use `set -e` so this had not been considered... but when I check the script now for this case: I only want to stop if the build fails so we don't run benchmarks on an old binary by accident while still allowing individual benchmarks to fail without skipping the remaining.